clang/lib/AST/Expr.cpp

```diff
@@ -658,7 +738,7 @@ std::string SYCLUniqueStableNameExpr::ComputeName(ASTContext &Context,
   llvm::raw_string_ostream Out(Buffer);
   Ctx->mangleTypeName(Ty, Out);
 
-  return Out.str();
+  return computeUniqueSYCLVXXName(Out.str(), true);
```

Comment:
the true means we are going to use our naming scheme all the time. even if Xilinx/AMD fpga is not a target.

Reply:
Add this as a comment, explaining why.